packages/permissions/src/hooks/use-is-authorized/use-is-authorized.spec.tsx

```diff
@@ -130,86 +130,90 @@ const testRender = ({
   );
 
 describe('when only one permission matches', () => {
-  describe('when it should match some permission', () => {
-    it('should indicate being authorized', () => {
-      const { queryByText } = testRender({
-        demandedPermissions: ['ManageCustomers', 'ManageOrders'],
-        allAppliedPermissions: [
-          {
-            name: 'canManageCustomers',
-            value: true,
-          },
-        ],
-        shouldMatchSomePermissions: true,
-      });
+  describe('General Permissions', () => {
+    describe('when it should match some permission', () => {
+      it('should indicate being authorized', () => {
+        const { queryByText } = testRender({
+          demandedPermissions: ['ManageCustomers', 'ManageOrders'],
+          allAppliedPermissions: [
+            {
+              name: 'canManageCustomers',
+              value: true,
+            },
+          ],
+          shouldMatchSomePermissions: true,
+        });
 
-      expect(queryByText('Is authorized: Yes')).toBeInTheDocument();
-    });
-  });
-  describe('when it should not match some permission', () => {
-    it('should indicate being not authorized', () => {
-      const { queryByText } = testRender({
-        allAppliedPermissions: [
-          {
-            name: 'canManageCustomers',
-            value: true,
-          },
-        ],
-        demandedPermissions: ['ManageCustomers', 'ManageOrders'],
-        shouldMatchSomePermissions: false,
+        expect(queryByText('Is authorized: Yes')).toBeInTheDocument();
       });
-
-      expect(queryByText('Is authorized: No')).toBeInTheDocument();
     });
-  });
-  describe('when it should not match an action right', () => {
-    it('should indicate being not authorized', () => {
-      const { queryByText } = testRender({
-        allAppliedPermissions: [
-          {
-            name: 'canManageCustomers',
-            value: true,
-          },
-        ],
-        allAppliedActionRights: [
-          {
-            group: 'products',
-            name: 'canEditPrices',
-            value: true,
-          },
-        ],
-        demandedPermissions: ['ManageCustomers'],
-        demandedActionRights: [{ group: 'products', name: 'PublishProducts' }],
-        shouldMatchSomePermissions: false,
-      });
+    describe('when it should not match some permission', () => {
```

Comment:
I don't get this part.. does this refer to the outcome or criteria?
if it is the criteria, then `when shouldMatchSomePermissions=false`